README.md

```diff
@@ -56,8 +56,8 @@ To unlock these features (optional):
 
 1. Download and install a beta version from GitHub. This is necessary because Marketplace does not allow publication of extensions that use proposed APIs.
 	- Go to https://github.com/intersystems-community/vscode-objectscript/releases
-	- Locate the beta immediately above the release you installed from Marketplace. For instance, if you installed `1.1.1`, look for `1.1.2-beta.1`. This will be functionally identical to the Marketplace version apart from being able to use proposed APIs.
-	- Download the VSIX file (for example `vscode-objectscript-1.1.2-beta.1.vsix`) and install it. One way to install a VSIX is to drag it from your download folder and drop it onto the list of extensions in the Extensions view of VS Code.
+	- Locate the beta immediately above the release you installed from Marketplace. For instance, if you installed `1.1.2`, look for `1.1.3-beta.1`. This will be functionally identical to the Marketplace version apart from being able to use proposed APIs.
```

Comment:
```suggestion
	- Locate the beta immediately above the release you installed from Marketplace. For instance, if you installed `1.2.0`, look for `1.2.1-beta.1`. This will be functionally identical to the Marketplace version apart from being able to use proposed APIs.
```

Reply:
Wouldn't the beta version be of the next pre-release? So it would be 1.3.0 with future beta's incrementing the patch number until 1.4.0 got released?

Reply:
I think we're still going to need the beta VSIXes for manual download and install because AFAIK we can't publish one in which proposed API has been enabled in package.json. Not even as a pre-release.

Reply:
That I know, my question was about the version numbers. I will just update this PR with the assumption that we're releasing 1.2.0 before adopting the pre-release mechanism.